client/elife-theme/src/elements/TextField.js

```diff
@@ -0,0 +1,44 @@
+import { css } from 'styled-components'
+import { th } from '@pubsweet/ui-toolkit'
+
+/**
+ * TODO
+ * this should be removed once it's implemented in pubsweet
+ * right now they don't have colorWarning
+ */
+const borderColor = ({ theme, validationStatus = 'default' }) =>
+  ({
+    error: theme.colorError,
+    success: theme.colorSuccess,
+    warning: theme.colorWarning,
+    default: theme.colorBorder,
+  }[validationStatus])
+
+/**
+ * TODO
+ *
+ * font-size & line-height might change to fontSizeBase in the future
```

Comment:
What is the specific event that this change depends on?

Reply:
Using hardcoded font-size for buttons seems like an odd decision. My assumption is that once we discuss these changes a little more with the design team, we will reach a conclusion to use `fontSizeBase` instead.

Reply:
The values for font size and line height are identical to that of `fontSizeBase`, so what's stopping us from using these variables right now?

Reply:
@jsms90 there was a discussion me & nick had (and this was discussed over several issues now). I think it's the same case as it is for button, if you go on invision and click "show resolved" from top right, then go to button font size you should see everything that's been discussed.

Reply:
I don't see anything in the comments that applies to this? It seems like that conversation was very specifically about the button

This is what @NickDuf says:
> The font-size and line height is typical to the button
> @MihailDunaev I think it would make sense for button fonts to have their own sizing.

I know you've had a long back and forth on this though. So perhaps there's other conversation elsewhere in Slack or something?

But it seems like that was motivated by the fact that the combination of font size and line height specified for the button was not covered by the typography combinations that exist in our theme?

Whereas this component specifies exactly the combination that corresponds to `fontSizeBase`/`fontLineHeightBase`. So I must be missing something, but why are we not using these variables?

Reply:
Yeah but component specification is hardcoded as well, I would assume it's a similar case.